tests/test_scheduled_tasks.py

```diff
@@ -109,3 +119,89 @@ def test_delete_invite_timestamp_issue(client: flask.testing.FlaskClient) -> Non
 def test_quarterly_usage(client: flask.testing.FlaskClient) -> None:
     """Test the quarterly_usage cron job."""
     quarterly_usage()
+
+
+def test_reporting_units_and_users(client: flask.testing.FlaskClient, fs: FakeFilesystem) -> None:
+    """Test that the reporting is giving correct values."""
+    # Create reporting file
+    reporting_file: pathlib.Path = pathlib.Path("doc/reporting/dds-reporting.csv")
+    assert not fs.exists(reporting_file)
+    fs.create_file(reporting_file)
+    assert fs.exists(reporting_file)
+
+    # Rows for csv
+    header: typing.List = [
+        "Date",
+        "Units using DDS in production",
+        "Researchers",
+        "Unit users",
+        "Total number of users",
+    ]
+    first_row: typing.List = [f"2022-12-10", 2, 108, 11, 119]
```

Comment:
`first_row: typing.List = [f"2022-12-10", 2, 108, 11, 119]`
the numbers (which are also in the initial csv file) add up if we ignore the users with _super admin_ role, but when the job runs it return a _Total number of users_ including them
maybe that's how you meant it, but I just wanted to mention it

Reply:
Hmm. Good point. I wanted to exclude the super admins in the file since it doesn't show the progress of the DDS in any way, but I didn't think of the number of users sum.  I'll change this.